[2025-02-24T15:45:51.361+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T15:45:51.404+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api.get_name scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-24T15:45:51.421+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api.get_name scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-24T15:45:51.422+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T15:45:51.462+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-02-10 00:00:00+00:00
[2025-02-24T15:45:51.472+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api', 'get_name', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp6eacmrig']
[2025-02-24T15:45:51.475+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=783) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T15:45:51.476+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask get_name
[2025-02-24T15:45:51.477+0000] {standard_task_runner.py:72} INFO - Started process 799 to run task
[2025-02-24T15:45:51.556+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api.get_name scheduled__2025-02-10T00:00:00+00:00 [running]> on host 99e935334142
[2025-02-24T15:45:51.688+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-24T15:45:51.690+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T15:45:51.693+0000] {python.py:240} INFO - Done. Returned value was: AirFlow
[2025-02-24T15:45:51.732+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T15:45:51.734+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api, task_id=get_name, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250224T154551, end_date=20250224T154551
[2025-02-24T15:45:51.812+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T15:45:51.858+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T15:45:51.863+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
